### PR TITLE
Basic filtering for normal case

### PR DESCRIPTION
**Pull request by geralt-encore**
Hey, I've done some work on basic filtering. Check it out, when you have some free time

### CODE REVIEW

**Comment by quentin23soleil** [comment]
Thanks @geralt-encore, I will have a look before the end of the week ;)

File: yeti/src/main/java/me/kentin/yeti/utils/SharePackageHelper.java [hunk 1]
```diff
@@ -105,6 +108,38 @@ public ApplicationType getApplicationType(Intent intent) {
         return ApplicationType.Other;
     }
 
+    public boolean isIntentAcceptable(ResolveInfo resolveInfo, List<ApplicationType> acceptableTypes) {
+        for (ApplicationType type : acceptableTypes) {
```
Comment (quentin23soleil) [inline yeti/src/main/java/me/kentin/yeti/utils/SharePackageHelper.java:115]: what is acceptableTypes is null (e.g the developer doesn't want to filter)

``` java
        yeti = Yeti.with(this);
        yeti.share(shareIntent);
```

Comment (quentin23soleil) [inline yeti/src/main/java/me/kentin/yeti/utils/SharePackageHelper.java:115]: ^ this would throw a NPE on acceptableTypes

**Comment by quentin23soleil** [comment]
@geralt-encore good job on that ;)
when you fix the NPE I'm happy to merge & release a new version :smile:

**Comment by geralt-encore** [comment]
@dommerq thanks, you are right =)
I've fixed it

**Pull request merged**